docs/public-api/public-api.v2.yaml

```diff
@@ -1109,6 +1112,12 @@ components:
               description: All measured values
               items:
                 type: number
+            stddev:
```

Comment:
This part of the API spec should probably be changed in the PR that also implements the change.

Reply:
This commit shouldn't be in here.